src/docs/product/sentry-basics/search/searchable-properties.mdx

```diff
@@ -4,64 +4,183 @@ sidebar_order: 10
-| release.stage | Restrict results to releases that are tagged with the matching release stage. See [Filtering Releases](/product/releases/usage/sorting-filtering/#filtering-releases) for supported values. |
-| transaction | Restrict results to events tagged with a URL template/job name. |
-| geo.country_code<br /> geo.region geo.city | Restrict results to events triggered by a geographic area. |
-| http.method<br /> http.referer<br /> http.url | Restrict results based on the HTTP request context. |
-| user.id user.email<br /> user.username<br /> user.ip | Restrict results to events affecting a given user. |
-| event.timestamp | Restrict results to events that occurred at the given timestamp. This filter can be passed twice to provide a range.<br /><br /> Events occurred on January 2, 2016: `event.timestamp:2016-01-02`<br /><br /> Events between 01:00 and 02:00 (UTC): `event.timestamp:>=2016-01-02T01:00:00 event.timestamp:<2016-01-02T02:00:00`<br /><br /> The following comparative operators are available: greater than (`>`), greater than or equal (`>=`), less than (`<`), less than or equal (`<=`) |
-| device.arch<br /> device.battery_level<br /> device.brand<br /> device.charging<br /> device.locale<br /> device.model_id<br /> device.name<br /> device.online<br /> device.orientation<br /> device.simulator<br /> device.uuid | Restrict results to events tagged with a specific device attribute. |
-| os.build<br /> os.kernel_version | Restrict results to events tagged with a specific operating system property. |
-| stack.abs_path<br /> stack.filename<br /> stack.function<br /> stack.module<br /> stack.package<br /> stack.stack_level<br /> stack.lineno | Restrict results to events with a matching stack property.<br /><br /> For Native SDK users, `stack.package` is the Native equivalent for `stack.module`. For more details about enhancing search, see the full documentation on [adding context and customizing tags](/platforms/native/enriching-events/). |
-| error.type<br /> error.value<br /> error.mechanism<br /> error.handled | Restrict results to events with a matching error property. |
-
-### Custom Tags
+Events are the underlying event data captured using Sentry SDKs — that is, errors and transactions.
+
+You can search by event properties in the following [sentry.io](https://sentry.io) pages:
+
+- Discover - in the query builder
+- Dashboards - within the widget builder, depending on dataset selection
+- Performance
+- Issues - as indicated in the table below
+- Alerts - when creating a metric alert
+
+When searching on event properties within the **Issues** page, the search will return any issue that has _one or more events_ matching the supplied event properties filter.
+
+### Release Properties
+
+In **Releases**, you can only search by by properties that include `release` in the key, but these properties are also searchable in the other pages of [sentry.io](https://sentry.io) listed above (under [Event Properties](#event-properties)).
```

Comment:
I think it might be better to list the 5 release fields here instead, since we may introduce fields that include relese eventually and there's also the `firstRelease` field

Reply:
Done!